src/main/java/frc/robot/subsystems/intake/IntakeCal.java

```diff
@@ -11,33 +11,36 @@ public class IntakeCal {
       INTAKE_PIVOT_D = Constants.PLACEHOLDER_DOUBLE;
 
   /** Motion profile max velocity and acceleration (deg per second) for intake pivot motor */
-  public static final double PIVOT_MAX_VELOCITY_DEG_PER_SECOND = Constants.PLACEHOLDER_DOUBLE,
-      PIVOT_MAX_ACCELERATION_DEG_PER_SECOND_SQUARED = Constants.PLACEHOLDER_DOUBLE;
+  public static final double PIVOT_MAX_VELOCITY_DEG_PER_SECOND = 30.0,
+      PIVOT_MAX_ACCELERATION_DEG_PER_SECOND_SQUARED = 30.0;
 
   /**
    * intake positions (degrees) where 90 degrees represents the intake at the shooter and 0 degrees
    * is down
    */
-  public static final double INTAKE_DEPLOYED_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE,
-      INTAKE_STOWED_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE,
-      INTAKE_SAFE_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE;
+  public static final double INTAKE_DEPLOYED_POSITION_DEGREES = 270.0,
+      INTAKE_STOWED_POSITION_DEGREES = 155.0,
+      INTAKE_SAFE_POSITION_DEGREES = 200.0; // exact safe position is 186.0
 
-  public static final double INTAKE_MARGIN_DEGREES = Constants.PLACEHOLDER_DOUBLE,
-      CONVEYOR_ZONE_THRESHOLD_DEGREES = Constants.PLACEHOLDER_DOUBLE;
+  public static final double INTAKE_MARGIN_DEGREES = 1.0,
+      CONVEYOR_ZONE_THRESHOLD_DEGREES = 190.0;
```

Comment:
How is this different from `INTAKE_SAFE_POSITION_DEGREES`?

Reply:
Added comment